app/templates/base.html

```diff
@@ -45,6 +45,16 @@
                     <li class="nav-item">
                         <a class="nav-link" href="/search">Search</a> 
                     </li>
+                    <li class="nav-item">
+                        <button onclick="getDailyJoke()" class="btn btn-link" style='text-decoration:none;'>get a joke</button>
+                        <script>
+                        function getDailyJoke() {
+                          fetch('/getjoke')
+                            .then(response => response.json())
+                            .then(data => alert(data.value.joke));
+                        }
+                        </script>
```

Comment:
It would be better if you could take it out to another function(a JavaScript one).

Reply:
I did